llvm/lib/Passes/PassRegistry.def

```diff
@@ -47,6 +47,7 @@ MODULE_PASS("always-inline", AlwaysInlinerPass())
 MODULE_PASS("annotation2metadata", Annotation2MetadataPass())
 MODULE_PASS("attributor", AttributorPass())
 MODULE_PASS("attributor-light", AttributorLightPass())
+MODULE_PASS("assign-guid", AssignUniqueIDPass())
```

Comment:
Looks like the surrounding items are sorted by alphabetical order, so this should come after "annotation2metadata"?